repipe

```diff
@@ -41,6 +41,20 @@ usage() {
     echo "validate          Validatei pipeline instead of set pipeline"
     echo "validate-strict   Validate pipeline with strict mode"
     echo "non-interactive   Run set-pipeline in non-interactive mode"
+    echo "url               Pipeline dashboard URL. Usage: ci/repipe url | xargs -L1 open"
+}
+
+show_pipeline_url() {
+	spruce merge --skip-eval pipeline.yml ${settings_file} > .deploy.yml
+	concourse_url=$(spruce json .deploy.yml | jq -r ".meta.url")
+	team=$(spruce json .deploy.yml | jq -r ".meta.team // \"main\"")
+	pipeline=$(spruce merge --skip-eval \
```

Comment:
Now need to perform `spruce merge` to get `meta.pipeline` but still be fast, so cherry-pick two required items